content/aws_marketplace.md

```diff
@@ -4,7 +4,7 @@ draft = false
 
 gh_repo = "chef-web-docs"
 
-aliases = ["/aws_marketplace.html"]
+aliases = ["/aws_marketplace.html", "/aws_ami.html"]
```

Comment:
We don't need this change, there's no current aws_ami page.

Reply:
They may have used it in marketing collateral. Would this work if they have something that links to that docs page (which doesn't exist)?

Reply:
I just checked, there was never an aws_ami page. This would work, but it would be better if links were fixed. Was there a page with that name in another section like automate or server?

Reply:
I don't know, but we can fix it next week in a different PR.